plugins/autosuspend.koplugin/_meta.lua

```diff
@@ -1,6 +1,6 @@
 local _ = require("gettext")
 return {
     name = "autosuspend",
-    fullname = _("Auto suspend"),
-    description = _([[Suspends the device after a period of inactivity.]]),
+    fullname = _("Auto power save"),
+    description = _([["Puts the device into standby, suspend or power off after specified periods of inactivity."]]),
```

Comment:
@zwim Could you please fix this?

Reply:
???

Reply:
It's either "" or [[]].

Reply:
That is, [["string"]] is the string `"string"`, not `string`.

Reply:
```suggestion
    fullname = _("Auto power save"),
    description = _([[Puts the device into standby, suspend or power off after specified periods of inactivity.]]),
```